data_sources/kafka_data_source.py

```diff
@@ -0,0 +1,61 @@
+from data_sources.data_source import DataSource
+from confluent_kafka import Consumer, KafkaError
+import time
+import json
+
+
+class KafkaDataSource(DataSource):
+    def __init__(self, kafka_endpoint, kafka_topic, s3_bucket, read_timeout_secs, batch_size = 1024):
+        super().__init__(s3_bucket)
+        self.kafka_endpoint = kafka_endpoint
+        self.kafka_topic = kafka_topic
+        self.read_timeout_secs = read_timeout_secs
+        self.batch_size = batch_size
+
+    def read_data_into_s3(self, file_size=1024 * 1024):
```

Comment:
A bit קטנוני, but let's do a constants file, and hold 1024 as a const instead of hard-coded value

Reply:
I think It's ok for a default value in a function